receiver/jmxmetricreceiver/integration_test.go

```diff
@@ -234,14 +234,13 @@ func (suite *JMXIntegrationSuite) TestJMXMetricViaOTLPReceiverIntegration() {
 		Password:     "cassandra",
 	}
 
-	extension := newJMXMetricExtension(logger, config)
-	require.NotNil(t, extension)
+	receiver := newJMXMetricReceiver(logger, config, otlpReceiver)
```

Comment:
This is confusing, why next consumer is `otlpReceiver` here?
It should be either exporter or a processor I think?

Reply:
I see a similar test with Prometheus receiver below. So how does the "cascade" receiver combination work?

Reply:
This was an unnecessary placeholder, and I've updated to use nil.  The complete receiver functionality isn't implemented yet, but when finished there will be an implicit otlp receiver created that receives the next consumer.